pages/residences-on-argis.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div>
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1 class="text-sm md:text-xl lg:text-2xl">
+          Places of Residence Visualised using ArcGIS Online
+        </h1>
+        <span
+          class="rounded-full border-r border-l border-gray-400 w-6 h-6 flex items-center justify-center ml-2"
+        >
+          <!-- icon by feathericons.com -->
+          <svg
+            aria-hidden="true"
+            class=""
+            data-reactid="266"
+            fill="none"
+            height="24"
+            stroke="#606F7B"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            stroke-width="2"
+            viewBox="0 0 24 24"
+            width="24"
+            xmlns="http://www.w3.org/2000/svg"
+            @click="showPageInfo()"
+          >
+            <line x1="12" y1="16" x2="12" y2="12"></line>
+            <line x1="12" y1="8" x2="12" y2="8"></line>
+          </svg>
+        </span>
+      </div>
+    </div>
+    <div>
+      <div class="embed-container-gismap">
+        <iframe
+          width="500"
+          height="400"
+          frameborder="0"
+          scrolling="no"
+          marginheight="0"
+          marginwidth="0"
+          title="Residence Location of Accused Scottish Witches (1563-1736)"
+          src="//edinuniv.maps.arcgis.com/apps/Embed/index.html?webmap=59369467c3a24fbe8c9fbe1a31bd707f&extent=-12.3301,51.9848,3.8967,57.4271&zoom=true&previewImage=false&scale=true&search=true&searchextent=true&legend=true&disable_scroll=false&theme=light"
```

Comment:
I couldn't get the map to load locally. Using the URL also doesn't appear to work but I'm not sure if it is meant to. Would you be able to look into this please Ruby? It might be that we can only embed from a particular URL so it might be blocked from rendering locally but I'm not sure as I don't know how this was set up. Thank you

Reply:
this is loading for me and the url itself loads, takes a second can you see if it still doesn't work for you?

Reply:
oops sorry, this was me not realising I have webGL disabled so it can't render. Looks good when I check it in another browser. Thanks Ruby